src/test/java/com/hermesworld/ais/galapagos/topics/service/impl/TopicServiceImplTest.java

```diff
@@ -780,22 +784,38 @@ public void testChangeDescOfTopic() throws Exception {
 	@Test
 	public void testDontResetDeprecationWhenTopicDescChanges() throws Exception {
 
+		SubscriptionService subscriptionService = mock(SubscriptionService.class);
+
+		TopicServiceImpl service = new TopicServiceImpl(kafkaClusters, applicationsService, topicNameValidator,
+				userService, topicConfig, eventManager);
+		ValidatingTopicServiceImpl validatingService = new ValidatingTopicServiceImpl(service, subscriptionService,
+				applicationsService, kafkaClusters, topicConfig);
+
+		UpdateTopicDto dto = new UpdateTopicDto("topic is now deprecated",LocalDate.of(2199,2,14),
```

Comment:
Missing space before comma (check formatter settings)

Reply:
Code is now formatted.